docs/getting-started/installation/installing-on-windows/installing-on-windows.rst

```diff
@@ -0,0 +1,68 @@
+Installing ASP.NET 5 On Windows
+===============================
+By `Steve Smith`_ | Originally Published: 1 June 2015 
+
+.. _`Steve Smith`: Author_
+
+ASP.NET 5 is now available on a variety of platforms, including Windows, OS X, and Linux. This article describes how to install ASP.NET 5 on Windows, showing both standalone installation as well as installation with Visual Studio 2015. 
```

Comment:
Remove "ASP.NET 5 is now available on a variety of platforms, including Windows, OS X, and Linux." to keep this focused on just Windows.